procedures/release-duty/index.rst

```diff
@@ -65,25 +65,21 @@ Email
 
 As ReleaseDuty you need to *subscribe* to certain mailing lists:
 
- * All types of sign-offs and approvals should go to `release signoff mailing list <https://mail.mozilla.org/listinfo/release-signoff>`__
```

Comment:
Can we revert this change. The mailinglists have been deprecated in favor of google groups - https://groups.google.com/a/mozilla.org/g/release-signoff/

Reply:
Thanks! I missed that somehow. Added back with google groups url.